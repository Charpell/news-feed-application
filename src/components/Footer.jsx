import React from 'react';

const date = new Date();
const year = date.getFullYear();

/**
 * Renders the footer
 * @returns {void}
 */
function Footer() {
  return (
    <footer className="footer navbar-default navbar-fixed">
      <div className="container">
        <p className="text-muted text-center">
          &copy;{year}, Made By Itua Akhator
        </p>
      </div>
    </footer>
  );
}

export default Footer;
