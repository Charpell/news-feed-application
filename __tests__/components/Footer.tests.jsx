import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../src/components/Footer';

describe('Footer Page', () => {
  it('renders a snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
