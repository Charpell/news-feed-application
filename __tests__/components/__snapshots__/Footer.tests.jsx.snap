// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Page renders a snapshot 1`] = `
<footer
  className="footer navbar-default navbar-fixed"
>
  <div
    className="container"
  >
    <p
      className="text-muted text-center"
    >
      Â© 2017, Made By Itua Akhator
    </p>
  </div>
</footer>
`;
